src/ipc.cpp

```diff
@@ -188,7 +189,6 @@ bool IPC::waitUntilAccepted(time_t postTime, int32_t timeout/*=-1*/)
     while (!(result = isEventAccepted(postTime)))
     {
         qApp->processEvents();
-        QThread::msleep(10);
```

Comment:
Sorry, I was wrong -> CPU actually spins to 100%. No problem though - I'll fix it and after we should be ok here.